aea/cli/utils/click_utils.py

```diff
@@ -205,43 +205,7 @@ def convert(self, value: str, param: Any, ctx: click.Context) -> str:  # type: i
             os.chdir(cwd)
 
 
-def registry_flag(
-    help_local: str = "Use only local registry.",
-    help_remote: str = "Use ony remote registry.",
-) -> Callable:
-    """Choice of one flag between: '--local/--remote'."""
-
-    def wrapper(f: Callable) -> Callable:
-        f = option(
-            "--local",
-            is_flag=True,
-            cls=MutuallyExclusiveOption,
-            help=help_local,
-            mutually_exclusive=["remote",],  # noqa: E231
-        )(f)
-        f = option(
-            "--remote",
-            is_flag=True,
-            cls=MutuallyExclusiveOption,
-            help=help_remote,
-            mutually_exclusive=["local",],  # noqa: E231
-        )(f)
-        return f
-
-    return wrapper
```

Comment:
Regarding your comment https://github.com/valory-xyz/open-aea/pull/103#discussion_r844502291

> I can see you didn't like this and therefore changed it.

It was more of a complexity concern. 

```python
        is_local: bool = False,
        is_remote: bool = False,
```

Is a bit hard to make sense of when we can easily define a single click param with 3 different flags `--local/--remote/--ipfs` and pass them as a string argument in a method the way I did here 

```python
@click.command()
@registry_flag()
@component_flag(wrap_public_id=True)
@click.pass_context
@check_aea_project
def add(
    click_context: click.Context,
    component_type: str,
    public_id: PublicId,
    registry: str,
) -> None:
```

Using boolean flags to define which registry to use seems a bit unnecessarily complicated to me.

For now I've just replaced the `--http` flag with `--remote` flag so the tests pass without changing anything, but if you still want to use the boolean flag approach it won't be hard to switch back. I only have one question, when passed the `--remote` flag which registry we should use http or ipfs?